TheMovieDb/New Group/Controllers/HomeViewController.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  ViewController.swift
+//  TheMovieDb
+//
+//  Created by Alex on 11/4/21.
+//
+
+import UIKit
+
+enum NetworkError: Error {
+    case badURL
+    case errorData
+}
+
+final class HomeViewController: UIViewController {
+    
+    @IBOutlet weak var randomMovie: UIImageView!
+    @IBOutlet weak var tableView: UITableView!
+    
+    static let trendingUrl: String = "/trending/movie/day?api_key=3f2d000acd208182b31eb1e5c2903ab8&language=en&region=US&page=1"
+    static let nowPlayingUrl: String = "/movie/now_playing?api_key=f6cd5c1a9e6c6b965fdcab0fa6ddd38a&language=en&region=US&page=1"
+    static let popularUrl: String = "/movie/popular?api_key=f6cd5c1a9e6c6b965fdcab0fa6ddd38a&language=en&region=US&page=1"
+    static let topRatedUrl: String = "/movie/top_rated?api_key=f6cd5c1a9e6c6b965fdcab0fa6ddd38a&language=en&page=1&region=US"
+    static let upcomingUrl: String = "/movie/upcoming?api_key=f6cd5c1a9e6c6b965fdcab0fa6ddd38a&language=en&region=US&page=1"
+    static let imagesBaseURL = "https://api.themoviedb.org/3"
+    private let movieSections = ["Trending" , "Now Playing", "Popular", "Top Rated", "Upcoming"]
+    private var movieList = [[Movie]]()
+    private let categoriesUrl: [String] = ["\(trendingUrl)",
+                                           "\(nowPlayingUrl)",
+                                           "\(popularUrl)",
+                                           "\(topRatedUrl)",
+                                           "\(upcomingUrl)"]
+    
+    var searchResultViewController = UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: "SearchResultViewController") as! SearchResultViewController
+    
+    var searchController: UISearchController!
+    
+    var filteredMovies: [Movie] = []
+    
+    var isSearchBarEmpty: Bool {
+      return searchController.searchBar.text?.isEmpty ?? true
```

Comment:
Nitpick: Please mind the correct indentation throughout the project; I believe that it is set to 4 spaces, so we need to be consistent with this across all the project. 🙂

Reply:
I've removed the spaces between the variables in that block.

Reply:
Thanks!